tracer/build/_build/Build.Steps.cs

```diff
@@ -373,6 +373,13 @@ IEnumerable<MSBuildTargetPlatform> ArchitecturesForPlatformForProfiler
         .After(CompileManagedLoader)
         .Executes(() =>
         {
+            // Removes previously generated files
+            // This is mostly to avoid forgetting to remove deprecated files (class name change or path change)
+            if (IsWin)
```

Comment:
Hmmm... is `IsWin` necessary 🤔 I don't _think_ it should be

Reply:
I'll be honest. It doesn't work on Mac. I've no idea why. I mean the source generators are ran, they can generate new files (if I add a new class for instance) but they won't regenerate the deleted files. 
I shamelessly ignored as I didn't want to spend too much time on it